packages/components/src/select-control/index.native.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * WordPress dependencies
+ */
+// import { withInstanceId } from '@wordpress/compose';
```

Comment:
We should remove the commented code 👍 
Same on `text-control/index.native.js` and `toggle-control/index.native.js `

Reply:
done ! :)